addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -671,6 +671,63 @@ QUnit.test('chat window: close on ESCAPE', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/closed']);
 });
 
+QUnit.test('focus next visible chat window when closing current chat window with ESCAPE', async function (assert) {
+    assert.expect(4);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [
+                {
+                    channel_type: "channel",
+                    id: 20,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "General",
+                    state: "open",
+                    uuid: "channel-20-uuid",
+                },
+                {
+                    channel_type: "channel",
+                    id: 21,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "other chat",
+                    state: "open",
+                    uuid: "channel-21-uuid",
+                },
+            ],
+        },
+    });
+    await this.start();
```

Comment:
Since test assumes screen is big enough to display 2 open chat windows at the same time, I would enforce mocked browser screen size in test:

```
start({
   env: {
      browser: {
         innerWidth: 1920,
      },
   },
});
```

Could add multi-line comment in 1st lines of test to explain provided mocked screen sizes, just in case we change width of chat windows in the future and if these tests fail, we know they have outdated assumptions and we also know how to adapt them accordingly. See test "open 2 different chat windows: enough screen width [REQUIRE FOCUS]" for format of multi-line comment.